CMakeLists.txt

```diff
@@ -1,7 +1,7 @@
 cmake_minimum_required(VERSION 3.13)
 # cmake_policy(SET CMP0076 OLD)
 
-set(FLAMESHOT_VERSION 11.0.0)
+set(FLAMESHOT_VERSION 11.0)
```

Comment:
Not semver compliant.

Reply:
I don't intend to follow semver... each release will bump the first number... patches will bump the second